### PR TITLE
Ignore log rules in BPF mode.

### PR DESCRIPTION
**Pull request by fasaxc**
## Description

Log rules are not suported but they were causing panics instead of being ignored.

Fixes https://github.com/projectcalico/calico/issues/4389

## Todos
- [x] Unit tests (full coverage)
- [ ] Backport
- [x] Release note

## Release Note

```release-note
Fix that, in eBPF mode, a Log rule would result in an error instead of being ignored.  Log rules are not supported but they should be ignored, not cause a failure.
```

### CODE REVIEW

**Review by neiljerram — APPROVED** [review]
LGTM

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and leave the branch after I've merged it.

**Pull request merged**
